index.html

```diff
@@ -41,6 +41,7 @@ <h2>Browser Features</h2>
 	<li><a href="./features/download/">Downloads</a></li>
 	<li><a href="./features/fonts.html">Fonts</a></li>
 	<li><a href="./features/autoconsent/">Cookie consent</a></li>
+	<li><a href="./features/autoconsent/banner.html">Cookie consent banners</a></li>
```

Comment:
Hm, I wonder if we should rename previous one to be sth like "Cookie consent popups" and this one to be "Cookie consent banners" or notifications?